_data/Diplomatie/Diplomatie.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Diplomatie
+institution: University Mcgill
+profile_pic: Diplomatie.jpg
+quote: “It is wrong always, everywhere, and for anyone, to believe anything upon insufficient evidence.” - William Kingdon Clifford
```

Comment:
The quote can't be more than 100 characters.

Also, please avoid using quotation marks ("), they can cause some errors in parsing.